.github/workflows/ci-test.yaml

```diff
@@ -85,7 +85,7 @@ jobs:
       - name: Validate storage migrations are serial
         run: scripts/validate_migrations.py --migrations storage/migrations
 
-  test-e2e:
+  test-e2e-localnet:
```

Comment:
Line 108 has `make test-e2e`, which refers to the target in the `Makefile`, which remains unchanged.
The name I changed here is mostly cosmetic; it only impacts how the stage is listed in CI results, AFAICT.

It would be perhaps a little nicer for consistency to also change the Makefile, but I don't want to go there, it's not the point of this PR. I can revert this change here though if you feel it's counterproductive.